src/main/java/mate/academy/spring/validation/EmailValidator.java

```diff
@@ -0,0 +1,17 @@
+package mate.academy.spring.validation;
+
+import java.util.regex.Pattern;
+import javax.validation.ConstraintValidator;
+import javax.validation.ConstraintValidatorContext;
+
+public class EmailValidator implements ConstraintValidator<Email, String> {
+    private static final String EMAIL_VALIDATOR = "^[A-Z0-9._%+-]+@[A-Z0-9.-]+\\.[A-Z]{2,6}$";
+
+    @Override
+    public boolean isValid(String email, ConstraintValidatorContext constraintValidatorContext) {
+        if (email == null) {
+            return false;
+        }
+        return Pattern.compile(EMAIL_VALIDATOR).matcher(email).matches();

```

Comment:
It is a good practice to compile your pattern only once in such services. It is better to create a constant of a type Pattern but not String